p/themes/base-theme/template.css

```diff
@@ -1769,7 +1756,7 @@ input:checked + .slide-container .properties {
 		bottom: 0;
 		left: 0;
 		right: 0;
-		cursor: pointer;
+		cursor: default;
```

Comment:
Is there somewhere else in the cascade that changes it? As written it looks like the line should simply be removed.

Reply:
`.dropdown-close` is a `<a>` and the default cursor would be the "hand" icon.
`cursor: default` = maus pointer (arrow).
Maybe you mix it with `cursor: auto`?

![grafik](https://user-images.githubusercontent.com/1645099/160289899-e8de670e-ac2f-4ae0-9dc9-ceedf01dbd57.png)

Reply:
No, I was just commenting on the diff.

Reply:
I checked. It is correct as it is

Reply:
I did not follow the whole discussion. What is the status? :-)

Reply:
All good probably.